### ISSUE TITLE
Incorrect behaviour when lava comes in contact with water source blocks

### ISSUE THREAD

**Issue by djmrFunnyMan** [Felicis/Water-Physics-Overhaul#13]
In vanilla when lava flows onto a "full" water block it will generate a stone block. This doesn't work with this mod however. Instead 2 things will happen
a) The water will turn into cobblestone
b) The water will completely disappear (when it's in a 1x1 hole usually)

https://github.com/Felicis/Water-Physics-Overhaul/assets/68905776/381fe8ef-d19e-4427-98f4-c285cfc6273b

**Comment by Felicis** [comment]
yeah, that's wrong. Has to do with water-lava interactions. Added it to the bug list. Will fix after fixing waterloggin #2  :)

**Comment by Felicis** [comment]
join the discussion to decide how water and lava should interact in WPO #20
